docs/advanced/images-adapters/cloudinary.mdx

```diff
@@ -0,0 +1,126 @@
+---
+sidebar_position: 1
+title: Cloudinary
+description:
+  Streamline media management and improve user experience by automatically
+  delivering images enhanced and optimized for every user.
+tags:
+  - Image Adapter
+  - Cloudinary
+  - Since version 2.20.x
+---
+
+import ContactLink from "@site/src/components/ContactLink";
+
+<p>{frontMatter.description}</p>
+
+## Installation
+
+Add the `cloudinary` module to your config file:
+
+```js title=".front-commerce.js"
+module.exports = {
+  name: "Front Commerce",
+  modules: [
+    "./node_modules/front-commerce/theme-chocolatine",
+    // highlight-next-line
+    "./node_modules/front-commerce/modules/cloudinary",
+    "./src",
+  ],
+};
+```
+
+## Configuration
+
+In order to use Cloudinary, you need to extend your
+[`src/config/images.js`](/docs/reference/configurations#configimagesjs) config.
+
+- `cloudName` (required): the Cloud Name of your Cloudinary environment.
+- `autoUploadMapping`: To attach an upload-preset to a mapped folder.
+- `transformations`: transformations which should be applied to the images.
+
+```js title="src/config/images.js"
+// Full configuration
+module.exports = {
+  cloudinary: {
+    cloudName: "my-cloud-name",
+    autoUploadMapping: [{ prefix: "/media", folder: "my-media-folder" }],
+    transformations: {
+      quality: "auto",
+      gravity: "auto",
+    },
+  },
+};
+```
+
+## Auto Upload Mapping
+
+You can automatically upload images to Cloudinary by mapping a folder to
+matching path.
+
+```js title="src/config/images.js"
+module.exports = {
+  cloudinary: {
+    cloudName: "my-cloud-name",
+    // highlight-start
+    autoUploadMapping: [
+      {
+        prefix: "/media", // the prefix path to match (for example, startsWith("/media"))
+        folder: "my-media-folder", // folder to upload to
+      },
+    ],
+    // highlight-end
+  },
+};
+```
+
+In cloudinary this can be configured by following the following guide:
+
+<iframe
+  src="https://scribehow.com/embed/Cloudinary__Auto_Upload_Mapping__don2zglETA6AUHheSjPTeA"
+  width="100%"
+  height="640"
+  allowfullscreen
+  frameborder="0"
+></iframe>
+
+## Transformations
+
+We currently only support the following transformations:
+
+- `quality`: The quality of the image to be delivered. defaults to `auto` (see
+  [docs](https://cloudinary.com/documentation/image_optimization#automatic_quality_selection_q_auto))
+- `gravity`: The part of the image to focus on. defaults to `auto` (see
+  [docs](https://cloudinary.com/documentation/resizing_and_cropping#control_gravity))
+
+Do not hesitate to <ContactLink>open an issue</ContactLink> if you need more
```

Comment:
**nitpick:** I'm not a fan of "open an issue", because this is a call to action for opening a conversation with a human on our side.

**suggestion:**  I'd suggest to remain with something simpler such as "ask us" or "contact us"